noir-projects/aztec-nr/tests/src/mock/test_note.nr

```diff
@@ -19,7 +19,7 @@ impl NoteInterface<TEST_NOTE_LENGTH> for TestNote {
     fn deserialize_content(fields: [Field; TEST_NOTE_LENGTH]) -> Self {
       Self {
         value: fields[0],
-        header: NoteHeader::empty(),
+        header: NoteHeader::default(),
```

Comment:
gonna be the annoying guy and ask that you add these changes in the migration_notes.md